store/mockstore/unistore.go

```diff
@@ -45,6 +45,13 @@ type mockStorage struct {
 	*tikv.KVStore
 }
 
+// NewMockStorage wraps tikv.KVStore as kv.Storage.
+func NewMockStorage(tikvStore *tikv.KVStore) kv.Storage {
```

Comment:
Why is it called 'mockstorage' while the tikv.KVStore is a real component?

Reply:
`KVStore` is real but it connects to an in-memory cluster 😄